chaos-bugbounty-list.json

```diff
@@ -9378,6 +9378,15 @@
       "domains":[
         "razer.com"
       ]
+    },
+    {
+      "name":"Mckesson",
+      "url":"https://hackerone.com/mckesson",
+      "bounty": false,
+      "domains":[
+        "mckesson.com",
+        "covermymeds.com"
```

Comment:
This scope looks different from their HackerOne profile. I don't see `covermymeds.com` and I see a lot more `mckesson` TLDs